packages/notifi-react-card/lib/components/intercom/SendMessageSection.tsx

```diff
@@ -31,6 +42,7 @@ export const SendMessageSection: React.FC<SendMessageSectionProps> = ({
       )}
     >
       <textarea
+        onKeyDown={handleKeypressDown}
```

Comment:
Since you are passing in the event in `handleKeypressDown`, should we update this so that it passes in the event? Something like

`onKeyDown={(e) => handleKeypressDown(e)}`